storybookjs/storybook
# CLI: Fix "reset" script removing .idea directory and add "cleanup" script

Issue: #18308 

## What I did

Fix the script to not delete `.idea`
Fix the wording in case when the user decides not to run the script
Add an additional script to delete `**/dist` only

## How to test

1. Create `.idea` dir in the root
1. Run `yarn bootstrap` and select `reset`
1. After script has run you should still see `.idea` in the root directory

1. Run `yarn bootstrap` and select `reset` script
1. Answer `No` or press Esc when asked "are you sure?"
1. It should say `Cleanup cancelled` instead of `problem is between keyboard and chair`

1. Run `yarn bootstrap` and select `cleanup` script
1. It should only delete `/dist` folders inside packages and leave `node_modules` and other files intact
